envi/tests/test_arch_arm.py

```diff
@@ -19,10 +19,9 @@
 
 
 ''' 
-
   This dictionary will contain all commands supported by ARM to test
   Fields will contain following information:
-  archBitMask, ophex, va, repr, flags, emutests
+  archBitMask, OPHEX, VA, repr, flags, emutests
```

Comment:
Done :)

Date: Fri, 3 Jun 2016 10:58:51 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Update unit tests for opcodes (#3)

In envi/tests/test_arch_arm.py:

>    This dictionary will contain all commands supported by ARM to test
>    Fields will contain following information:
> -  archBitMask, ophex, va, repr, flags, emutests
> +  archBitMask, OPHEX, VA, repr, flags, emutests

no thank you.  please keep the CAPITALS to a minimum.  just an aesthetic, but something i feel strongly about.


—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.